src/js/components/landing/blocks/LandingRequirementsModal.jsx

```diff
@@ -42,6 +43,69 @@ export default class LandingRequirementsModal extends React.Component {
 			gtas = <strong>*Note: The GTAS Submission Window is currently open. You cannot certify or re-certify until after the window closes on {date}.</strong>
 		}
 
+		let header = "You'll need the following files in order to complete your submission";
+		let body = <div>
```

Comment:
We may want to further modularize the body of the modal into a FabsRequirementsModalBody component and a DabsRequirementsModalBody component, rather than putting it all in the LandingRequirementsModal component.